docs/tools/testing/testing_greentea.md

```diff
@@ -67,58 +71,190 @@ The build system looks for an `mbed_app.json` file in your shared project files
 
 If you need to test with multiple configurations, then you can use the `--app-config` option. This overrides the search for an `mbed_app.json` file and uses the configuration file that you specify for the build.
 
-#### Running tests
+### Writing your first test
 
-You can run automated tests through Mbed CLI.
+You can write tests for your own project or add more tests to Mbed OS. You can write tests by using the [Greentea client](https://github.com/ARMmbed/mbed-os/tree/master/features/frameworks/greentea-client), and the [UNITY](https://github.com/ARMmbed/mbed-os/tree/master/features/frameworks/unity) and [utest](https://github.com/ARMmbed/mbed-os/tree/master/features/frameworks/utest) frameworks, which are located in `/features/frameworks`.
 
-The testing process requires that tests be built and that a test specification JSON file that describes these available tests exists. See the [test specification format](https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-greentea#test-specification-json-formatted-input).
+Let's write your first test. Use Mbed CLI to create a new project:
+
+```
+$ mbed new first-greentea-test
+```
 
-The Greentea tool handles the actual testing process. To read more about this tool, please visit its [GitHub repository](https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-greentea).
+As specified above, there is a convention where all tests live in the `TESTS/` directory. In the `first-greentea-test` folder create a folder `TESTS/tests/simple-test/`.
 
-#### Writing tests
+![Exporting to Make](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/test01.png)
```

Comment:
We should promote working with Mbed CLI within it's own documentation where possible. Either this is titled incorrectly, or it should be removed (I can't see the image content)

Reply:
Hmm, I think I copied something wrong there. The description just shows the tree structure, nothing to do with make. My bad :-)